### DISCUSSION TITLE
How can I create three navigations based on the same model?

### DISCUSSION THREAD

**Discussion by shebaoting** [filamentphp/filament#12460] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

I have a student model. But each student's class is different. I want to use navigation on the left side to distinguish classes. For example, there are three navigations on the left side, namely, Class One, Class Two, and Class Three. Do I need to create three filament-resources based on the student model? Is there a better way?

**Comment by dissto** [comment]
[Custom navigation](https://filamentphp.com/docs/3.x/panels/navigation#overview)?
